tests/src/Core/Console/AutomaticInstallationConsoleTest.php

```diff
@@ -270,9 +270,13 @@ public function testWithArguments()
 
 	/**
 	 * @runTestsInSeparateProcesses
+	 * @preserveGlobalState disabled
 	 */
 	public function testNoDatabaseConnection()
 	{
+		// TODO DBA mocking for whole console tests make this test work again
```

Comment:
if you use three slashes  and and  @ (`/// @TODO some to do text`) it will be added to the doxygen documentation as a todo

Reply:
thanks for the hint